packages/cli/src/executions/execution.service.ts

```diff
@@ -478,4 +483,17 @@ export class ExecutionService {
 
 		return await this.executionRepository.stopDuringRun(execution);
 	}
+
+	async enrichExecutionSummariesWithScopes(user: User, summaries: ExecutionSummary[]) {
```

Comment:
Maybe simply `addScopes`? That it's execution summaries is clear from the signature.